### PR TITLE
Open-Elevation (Independent Publisher)

### PR DESCRIPTION
**Pull request by fordosa90**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by fordosa90** [comment]
image

**Comment by fordosa90** [comment]
image

File: independent-publisher-connectors/Open-Elevation/readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Publisher: Fördős András
+
+## Pre-requisites
+The service is available as a public and free service, hence no pre-requisites exists. That said, since it is for public use, it welcomes donations to keep the servers running: [https://www.open-elevation.com/](https://www.open-elevation.com/)
+
+## Supported Operations
+### Lookup
+Returns ("looks up") the elevation at one or more (latitude,longitude) points.
+
```
Comment (vmanoharas) [inline independent-publisher-connectors/Open-Elevation/readme.md:15]: Please add Obtaining Credentials section, which is missing.

**Comment by fordosa90** [comment]
@vmanoharas

**Comment by fordosa90** [comment]
@vmanoharas  see PR from 2 months ago

File: independent-publisher-connectors/Open-Elevation/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Open-Elevation",
+    "description": "Open-Elevation is a free and open-source alternative to the Google Elevation API and similar offerings.",
```
Comment (vmanoharas) [inline independent-publisher-connectors/Open-Elevation/apiDefinition.swagger.json]: Can you please remove API keyword from description

**Review by vmanoharas — COMMENTED** [review]
Hi @fordosa90 

Can you please resolve review comments

**Review by vmanoharas — COMMENTED** [review]
Hi @fordosa90, 

Please accept my apology for not submitting my review added. Can you please resolve them. Apologies again.

**Comment by fordosa90** [comment]
@vmanoharas I have left comments regarding your notes, and would require feedback before performing any change - if needed. Could you please review

**Comment by vmanoharas** [comment]
> @vmanoharas I have left comments regarding your notes, and would require feedback before performing any change - if needed. Could you please review

Hi @fordosa90, 

I am not able to find your comments, could you please add, if you haven't added comments.

**Comment by fordosa90** [comment]
@vmanoharas  as stated in above paragraph, the service is public, hence there is no credential
as far as I know, there is no definitive guide for these readme.md files but I see other ones in the repository where the service is public and not having a special section in the markdown to repeat, that there are no credentials.
do you require to have a specific paragraph to state in addition to the "public" notion the no need for credentials?

 the "Google Elevation API" is a trademarked product and service name, over which I don't have ownership. Removing the API would affect brand. Do you want to proceed?

**Comment by vmanoharas** [comment]
> @vmanoharas as stated in above paragraph, the service is public, hence there is no credential as far as I know, there is no definitive guide for these readme.md files but I see other ones in the repository where the service is public and not having a special section in the markdown to repeat, that there are no credentials. do you require to have a specific paragraph to state in addition to the "public" notion the no need for credentials?
> 
> the "Google Elevation API" is a trademarked product and service name, over which I don't have ownership. Removing the API would affect brand. Do you want to proceed?

Hello @fordosa90, 

Thank you for providing the details. 

Regarding, please add obtaining credential section and mention it as NA. 

Regarding API keyword, please remove this word, they may affect for automation tools.

**Comment by fordosa90** [comment]
@vmanoharas updates pushed

**Review by vmanoharas — APPROVED** [review]
Hello @fordosa90, 
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by vmanoharas** [comment]
Hello @fordosa90 ,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Pull request merged**